decisions/wc_politfaction_decisions.txt

```diff
@@ -186,6 +182,8 @@ decisions = {
 		}
 	}
 	leave_horde_political_faction = {
+		is_high_prio = yes
```

Comment:
Are you sure it is should be in high prio by default?

Reply:
One guy told me he's trapped in the Alliance and can't leave it. So, ye, I'm sure.

Reply:
weird flex but ok